util/reggen/gen_dv.py

```diff
@@ -53,6 +53,16 @@ def rcname(esc_if_name: str, r: Union[Register, MultiRegister]) -> str:
     return '{}_reg_{}'.format(esc_if_name, r.name.lower())
 
 
+def alias_rcname(esc_if_name: str,
+                 r: Union[Register, MultiRegister]) -> Optional[str]:
+    '''Get the name of the dv_base_reg subclass for this alias register'''
+    alias_target = getattr(r, 'alias_target', None)
```

Comment:
This is a bit surprising! Don't we unconditionally have an `alias_target` attribute (which might be None)?

Reply:
It is surprising, yes. Let me have a look at this again why put this there...

Reply:
Indeed this is now not needed anymore (probably also a leftover from a previous iteration).